infra/Pos/Slotting/MemState/Holder.hs

```diff
@@ -46,8 +45,18 @@ askSlottingTimestamp  = fst <$> askSlotting
 -- MonadSlotsData implementation
 ----------------------------------------------------------------------------
 
-instance MonadIO m =>
-         MonadSlotsData (SlottingHolder m) where
+data SlotsDataRedirectTag
+
+type SlotsDataRedirect =
+    Ether.TaggedTrans SlotsDataRedirectTag IdentityT
+
+runSlotsDataRedirect :: SlotsDataRedirect m a -> m a
+runSlotsDataRedirect = coerce
```

Comment:
I see this `Redirect` suffix is almost everywhere. What does it mean? Dunno if it makes sense to document `Redirect` stuff in all places.